xbmc/win32/Win32DelayedDllLoad.cpp

```diff
@@ -70,5 +70,5 @@ FARPROC WINAPI delayHookFailureFunc (unsigned dliNotify, PDelayLoadInfo pdli)
 }
 
 // assign hook functions
-PfnDliHook __pfnDliNotifyHook2 = delayHookNotifyFunc;
-PfnDliHook __pfnDliFailureHook2 = delayHookFailureFunc;
```

Comment:
@FernetMenta 
This throws an compiler error with VS2015 Update 3. Who can I ping about this? I guess @afedchin does not have the time to look into right?